po/cs.po

```diff
@@ -14420,8 +14418,9 @@ msgid "Match existing transactions"
 msgstr "Zvolte odpovídající existující účetní položku"
 
 #: gnucash/gtkbuilder/assistant-qif-import.glade:1269
+#, fuzzy
 msgid "_Imported transactions needing review"
-msgstr "První část importované účetní položky"
```

Comment:
@fellen look how many times was this translation used, would be nice to detect all of this and remove from everywhere, so it will be clean.